client/components/footer/footer.js

```diff
@@ -24,7 +24,7 @@ export default class Footer extends React.Component {
                   footer-link-${index + 1}`}
                     target="_blank"
                     rel="noreferrer noopener"
-                    key={link.url}
+                    key={link}
```

Comment:
Can we set ``key=index``? 
As link is an object we will again get unique key props error. 
What do you think?